morphsnakes/include/ndimage.h

```diff
@@ -97,15 +97,15 @@ class NeighborOffsets<2>
 {
 public:
     static const int num_neighbors = 9;
-    
+
     typedef Position<2>::Coord Coord;
     typedef std::array<Coord, num_neighbors> CoordOffsets;
     typedef std::array<int, num_neighbors> LinearOffsets;
     
     static constexpr CoordOffsets coord_offsets = {
-        {{-1, -1}, {-1, 0}, {-1, 1},
-        {0, -1}, {0,0}, {0, 1},
-        {1, -1}, {1, 0}, {1, 1}}
+         {Coord {-1, -1}, Coord {-1, 0}, Coord {-1, 1},
+          Coord {0, -1}, Coord {0, 0}, Coord {0, 1},
+          Coord {1, -1}, Coord {1, 0}, Coord {1, 1}}
```

Comment:
Is there any reason for explicitly adding the constructor `Coord`? The type `CoordOffsets` is defined in such a way that this should be the default behavior. Did you find any problem with it?

Reply:
yes, (it is a while ago but I think that) I could not compile it on my computer...